src/components/ViewEditSite/ViewEditSite.jsx

```diff
@@ -0,0 +1,539 @@
+import { React, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import './ViewEditSite.css';
+import { Container, Col, Row } from 'react-bootstrap';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+import { Link } from 'react-router-dom';
+import states from 'states-us';
+import { TLPBackend } from '../../common/utils';
+
+const s = states.filter(x => !x.territory);
+const abbrev = s.map(x => x.name);
+// remove district of columbia
+abbrev.splice(8, 1);
+const options = abbrev.map(x => ({ label: x, value: x }));
+
+const schema = yup
+  .object({
+    siteName: yup.string().required(),
+    addressStreet: yup.string().required(),
+    addressApt: yup.string(),
+    addressCity: yup.string().required(),
+    addressState: yup.string().required(),
+    addressZip: yup.number().required(),
+    primaryFirstName: yup.string().required(),
+    primaryLastName: yup.string().required(),
+    primaryTitle: yup.string(),
+    primaryEmail: yup.string().required(),
+    primaryPhone: yup.string().required(),
+    secondaryFirstName: yup.string(),
+    secondaryLastName: yup.string(),
+    secondaryTitle: yup.string(),
+    secondaryEmail: yup.string(),
+    secondaryPhone: yup.string(),
+    notes: yup.string(),
+  })
+  .required();
+
+const ViewSite = ({ siteId }) => {
+  const { register, handleSubmit } = useForm({
+    resolver: yupResolver(schema),
+    delayError: 750,
+  });
+
+  const siteInfo = JSON.parse(localStorage.getItem('siteInfo'));
+  const [edit, setEdit] = useState(false);
+  const [areaName, setAreaName] = useState('');
+
+  const abbrevState = s.filter(x => x.name === siteInfo.addressState)[0].abbreviation;
```

Comment:
I opened the area dropdown and clicked on a site, and got this error. This also happens when I go to an AreaView, and then press "View info"

![image](https://user-images.githubusercontent.com/57849152/165683998-bc009836-526d-405c-9d4e-6d6bd2ddde90.png)